.github/workflows/go.yml

```
@@ -0,0 +1,22 @@
+name: Go
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Set up Go
+      uses: actions/setup-go@v4
+      with:
+        go-version: '1.20'
```

Can we rename this to ...

```suggestion
        go-version: 'stable'
```

---

Yep - I didn't particularly mean to use 1.20